standup/status/models.py

```diff
@@ -30,6 +32,10 @@
 ])
 
 
+CLEANER = Cleaner(tags=[])
+LINKER = Linker(callbacks=[trim_urls, nofollow])
```

Comment:
Pretty sure we should have been doing "nofollow" all along. So I took the time in this PR to add that.